R/fda-table_33.R

```diff
@@ -0,0 +1,62 @@
+# Pre-processing
+
+library(tidyverse)
+library(scda)
+library(tern)
+
+# Safety population
+adsl <-
+  scda::synthetic_cdisc_dataset("rcd_2022_06_27", "adsl") %>%
+  dplyr::filter(SAFFL == "Y") %>%
+  dplyr::select(STUDYID, USUBJID, SAFFL, ARM, ACTARM)
+
+# Vital signs
+advs <-
+  scda::synthetic_cdisc_dataset("rcd_2022_06_27", "advs") %>%
+  dplyr::filter(PARAMCD %in% c("DIABP", "SYSBP") & SAFFL == "Y" & AVISITN >= 1) %>% # nolint
+  dplyr::select(STUDYID, USUBJID, ARM, ACTARM, PARAMCD, PARAM, AVAL, AVALU, AVISITN, AVISIT, SAFFL) %>% # nolint
+  dplyr::group_by(USUBJID, PARAMCD) %>%
+  dplyr::mutate(
+    maxDIABP = if_else(PARAMCD == "DIABP", max(AVAL), NA_real_),
+    maxSYSBP = if_else(PARAMCD == "SYSBP", max(AVAL), NA_real_)
+  ) %>%
+  dplyr::mutate(
+    SBP90 = (PARAMCD == "SYSBP" & maxSYSBP < 90),
+    DBP60 = (PARAMCD == "DIABP" & maxDIABP < 60),
+  ) %>%
+  formatters::var_relabel(
+    SBP90 = "SBP <90",
+    DBP60 = "DBP <60"
+  )
+
+# Build layout
+lyt <-
+  rtables::basic_table(show_colcounts = TRUE) %>%
+  rtables::split_cols_by("ARM") %>%
+  tern::count_patients_with_flags(
+    var = "USUBJID",
+    flag_variables = var_labels(advs[, c("SBP90", "DBP60")]), # nolint
+    table_names = "bp"
```

Comment:
This can be excluded - if you're only using a function once in a table you shouldn't need to define `table_names` unless you're using the table structure later.
```suggestion
    flag_variables = var_labels(advs[, c("SBP90", "DBP60")])
```